README.md

```diff
@@ -57,21 +57,21 @@ git clone https://github.com/microsoft/WinUI-Gallery.git
 
 ### 3. Open .sln with Visual Studio and Build!
 ```shell
-.\WinUIGallery.sln
+WinUIGallery.DesktopWap.sln
 ```
-Ensure that `WinUIGallery Packaged` is set as the startup project in Visual Studio.
+Ensure that the `WinUIGallery.DesktopWap.Packaged` is set as the startup project in Visual Studio.

```

Comment:
Why should one use the DesktopWap solution when the WinUIGallery solutino works just fine?

Reply:
*WinUIGallery.sln* won't load the *WinUIGallery.DesktopWap.Packaged* project. Is it different on your end?

Reply:
I'm not using the `DesktopWap` either

Reply:
Can I update the instructions to just simply open *WinUIGalley.sln*?

Reply:
I think pointing out the startup project is also important here but besides that sure